### ISSUE TITLE
nvda does not detect progressbar when copying files in win 8.1

### ISSUE THREAD

**Issue by kaveinthran** [nvaccess/nvda#7552]
### Steps to reproduce:
> A list of the steps you take to demonstrate the problem. 
>
> Example:
>
> 1. Open Chrome
> 2. Browse to www.google.com
> 3. Type "Hello"
> 4. Notice an error sound when enter is pressed.

> Please also remember to attach a zip of any files required to reproduce the issue.

### Expected behavior:
> Tell us what should happen.

### Actual behavior:
> Tell us what happens instead.

### System configuration:
NVDA version:
> Example: next-14027,c80e529f

NVDA Installed or portable:

Other information:
> Example: Running in a VM

Windows version:
> Example: Windows 10 Version 1607 Build 14393.1066

Name and version of other software in use when reproducing the issue:

### Other questions:

Does the issue still occur after restarting your PC?

Have you tried any other versions of NVDA?
> Please list them and the result
Step to reproduce

1. copy files or folder from windows explorer and paste it in another drive, make sure the folder is big, to watch the progressbar
2. expected result is to have nvda progress indicator to fire up, 
3. actual result: nothing happen, but the progressbar can be tracked by reading the window title, which is not autonomous and needs pressing win+t occationally

thanks

**Comment by Adriani90** [comment]
@kaveinthran actually you should be able to read the progress bar either with nvda+b or by using object navigation. Can you confirm?

**Comment by Adriani90** [comment]
I am closing this issue as abandoned due to (I) missing updates from the initial author and (II) support for Windows 8 having ended years ago. In Windows 10 and above this issue is not reproducible.

**Comment by CyrilleB79** [comment]
> I am closing this issue as abandoned due to (I) missing updates from the initial author and (II) support for Windows 8 having ended years ago. In Windows 10 and above this issue is not reproducible.

@Adriani90, I can see that you are closing issues related to windows 8 / 8.1. According to NVDA's user guide, the supported OSs are:
> Operating Systems: all 32-bit and 64-bit editions of Windows 7, Windows 8, Windows 8.1, Windows 10, Windows 
11, and all Server Operating Systems starting from Windows Server 2008 R2.

Windows 8 and 8.1 are now unsupported by Microsoft for many years (as Windows 7 is) and is also much more less popular than Windows 7. So such issues will probably be very low priority. But unless NVAccess says otherwise, they are still supported by NVDA; end of support for NVDA for Windows 7, 8 and 8.1 will probably occur for 2024.1 due to Python upgrade in any case.

For the current issue though, the first closing reason (no news from issue's the author) still stands.
